tensorflow/tools/docker/Dockerfile.gpu

```diff
@@ -58,6 +58,9 @@ COPY notebooks /notebooks
 # We just add a little wrapper script.
 COPY run_jupyter.sh /
 
+# We depend on libcupti.so
+ENV LD_LIBRARY_PATH /usr/local/cuda/extras/CUPTI/lib64:$LD_LIBRARY_PATH
```

Comment:
Are we sure libcupti is installed in the docker image?
Maybe this should be upstreamed into nvidia docker?